changelogs/fragments/1182_ansible_version.yml

```
@@ -0,0 +1,2 @@
+breaking_changes:
+- The collection now requires at least ansible-core 2.11.0. Ansible 2.9 and before, and ansible-base versions are no longer supported.
```

```suggestion
- The collection now requires at least ansible-core 2.11.0. Ansible 3 and before, and ansible-base versions are no longer supported.
```
Maybe it makes sense to mention Ansible 3 here, since the community Ansible 2.10 and Ansible 3 packages both used ansible-base.